### ISSUE TITLE
Can't break out Google Analytics metrics by Date:Month

### ISSUE THREAD

**Issue by mazameli** [metabase/metabase#11489]
If I try to do something like Users per Date:Month on our stats instance with Google Analytics, I get the error, `Cannot read property 'cols' of undefined`.

The screen looks like this:

![image](https://user-images.githubusercontent.com/2223916/70566401-d8f8ca00-1b48-11ea-96b0-c8d7df9730b3.png)

**Comment by mazameli** [comment]
On our main Google Analytics dashboard on our stats instance, I'm seeing the same error in the JavaScript console:

```
Error getting setting graph.x_axis.title_text TypeError: Cannot read property 'cols' of undefined
    at Object.getDefault (app-main.bundle.js?1a7af45b230b404472ca:5)
    at g (app-main.bundle.js?1a7af45b230b404472ca:5)
    at t.getComputedSettings (app-main.bundle.js?1a7af45b230b404472ca:5)
    at h (app-main.bundle.js?1a7af45b230b404472ca:5)
    at t.value (app-main.bundle.js?1a7af45b230b404472ca:5)
    at t.value (app-main.bundle.js?1a7af45b230b404472ca:5)
    at c.performInitialMount (vendor.bundle.js?1a7af45b230b404472ca:1)
    at c.mountComponent (vendor.bundle.js?1a7af45b230b404472ca:1)
    at Object.mountComponent (vendor.bundle.js?1a7af45b230b404472ca:1)
    at Object.mountComponent (backend.js:30)
```

**Comment by Wongmat** [comment]
Getting the exact problem here. Trying to convert the question into a native query puts the app into a loop with the same error: 

```
Could not load worker DOMException: Failed to construct 'Worker': Access to the script at 'blob:https://mb.bowtie.com.hk/efebcc0a-9af4-489b-8954-1656ad1aa5a2' is denied by the document's Content Security Policy.
    at r (https://mb.bowtie.com.hk/app/dist/vendor.bundle.js?f4999ff7ddee79f5a90a:1:1982823)
    at c.$createWorkerFromOldConfig (https://mb.bowtie.com.hk/app/dist/vendor.bundle.js?f4999ff7ddee79f5a90a:1:1983658)
    at new c (https://mb.bowtie.com.hk/app/dist/vendor.bundle.js?f4999ff7ddee79f5a90a:1:1983020)
    at l.createWorker (https://mb.bowtie.com.hk/app/dist/vendor.bundle.js?f4999ff7ddee79f5a90a:1:2047674)
    at p.$startWorker (https://mb.bowtie.com.hk/app/dist/vendor.bundle.js?f4999ff7ddee79f5a90a:1:1813185)
    at p.$onChangeMode (https://mb.bowtie.com.hk/app/dist/vendor.bundle.js?f4999ff7ddee79f5a90a:1:1812424)
    at p.setMode (https://mb.bowtie.com.hk/app/dist/vendor.bundle.js?f4999ff7ddee79f5a90a:1:1811946)
    at t.value (https://mb.bowtie.com.hk/app/dist/app-main.bundle.js?f4999ff7ddee79f5a90a:5:2064592)
    at e.notifyAll (https://mb.bowtie.com.hk/app/dist/vendor.bundle.js?f4999ff7ddee79f5a90a:1:1289748)
    at d.close (https://mb.bowtie.com.hk/app/dist/vendor.bundle.js?f4999ff7ddee79f5a90a:1:2231106)
```

Creating a brand new native query with the same parameters gives this error: 
<img width="582" alt="Screenshot 2019-12-23 at 2 45 55 PM" src="https://user-images.githubusercontent.com/31556051/71340856-09058b80-2593-11ea-96d9-4b9aeb93187b.png">

Seems to be something to do with how `ga:yearMonth` is parsed as using any other dimension in place gets rid of the issue.

**Comment by tiwariayush** [comment]
Hi, are there any progress on this issue ? We have a lot of questions as well as dashboards which are giving error which were written with using `Date::Month`.

**Comment by camsaul** [comment]
Fixed by #11681
